Editor/ModelImporter_GLTF.cpp

```diff
@@ -2227,7 +2227,7 @@ void Import_Extension_VRM(LoaderState& state)
 				for (size_t boneGroup_index = 0; boneGroup_index < boneGroups.ArrayLen(); ++boneGroup_index)
 				{
 					const auto& boneGroup = boneGroups.Get(int(boneGroup_index));
-					SpringComponent component;
+					SpringComponent component{};
```

Comment:
Was this added because there was uninitialized member in springcomponent? I added missing initialization to member, can this now be removed if that was the reason?

Reply:
it complained about entity not being set. I've updated the PR, it now defaults entity to -1 in the struct and removes the `{}`